src/app/(Authentication)/OTPFlow/OTPVerify/index.tsx

```diff
@@ -58,9 +57,7 @@ export default function OTPFlow() {
       <View style={styles.contentContainer}>
         <View style={styles.instructionContainer}>
           <Text style={styles.headerText}>Enter verification code.</Text>
-          <Text style={styles.instructionText}>
-            We've just emailed it to you at {email}
-          </Text>
+          <Text style={styles.instructionText}>We've sent it to {email}</Text>
```

Comment:
Apply a 100% width so it fills the content container and has the same width as the OPT container below.